next.config.js

```diff
@@ -131,7 +131,7 @@ const appEnvironments = {
     name: 'DEV5',
     showName: true,
     airtable: {
-      apiBase: prodAirtableApiBase,
+      apiBase: liveAirtableApiBase,
```

Comment:
Not sure how to go about updating the production airtable

Reply:
try the airtable dump: npm run airtable:dump